test/spec/reader.js

```diff
@@ -28,7 +38,7 @@ describe('Reader', function() {
       var xml = '<props:complexAttrs xmlns:props="http://properties"></props:complexAttrs>';
 
       // when
-      reader.fromXML(xml, rootHandler, function(err, result, context) {
+      read(reader, xml, rootHandler, function(err, result, context) {
```

Comment:
Please transform the tests into `async/await` style ones:


```
it('should do stuff', async function() {
  
   // when
   var {
      element,
      parseContext
   } = await reader.fromXML(...);

   ...
});
```

Reply:
regex FTW